File: responses/4N53A7-Z4R/sum_of_integers_in_a_string.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(S):
    S = S.split(',')
    result = 0
    for num in S:
        if '+' in num:
            num = num.replace('+','')
        result += int(num)
    return result

File: responses/YJ7PGK-HKS/sum_of_integers_in_a_string.rb
# you can write to stdout for debugging purposes, e.g.
# puts "this is a debug message"

def solution(a)
  answer = 0
  counter = Hash.new { 0 }
  
  a.each do |value|
    counter[value] += 1
    
    answer = counter[value] if counter[value] > answer
  end
  
  return answer
end
